FESS-GUI/flywheeloperation.cpp

```diff
@@ -1,58 +1,258 @@
+// Custom Libraries
 #include "flywheeloperation.h"
-#include <cmath>
-#include <QTime>
-#include <QtMath>
 
-FlywheelOperation::FlywheelOperation()
+FlywheelOperation::FlywheelOperation(Interface *inter)
```

Comment:
The nice thing about using a constructor like this is, we can change the interface as needed. Any object that is a subclass of Interface can be used. Currently these are Demo and Serial.